src/game/Spell.cpp

```diff
@@ -2685,6 +2685,19 @@ void Spell::cast(bool skipCheck)
                 AddPrecastSpell(25771);                     // Forbearance
             break;
         }
+		case SPELLFAMILY_DRUID:
+		{
+			switch (m_spellInfo->Id)
+			{
+			case 24866:
+				if (!m_caster->IsInFeralForm())
```

Comment:
This is still wrong. You have to check if the current form is FORM_CAT and not if it is any form.